x/btcstaking/keeper/btc_delegations.go

```diff
@@ -111,6 +136,15 @@ func (k Keeper) addCovenantSigsToBTCDelegation(
 		activeEvent := types.NewEventPowerDistUpdateWithBTCDel(event)
 		btcTip := k.btclcKeeper.GetTipInfo(ctx)
 		k.addPowerDistUpdateEvent(ctx, btcTip.Height, activeEvent)
+	} else if btcDel.HasCovenantQuorums(params.CovenantQuorum) && !btcDel.HasInclusionProof() {
```

Comment:
If conditions in L125 and L139 could be consolidated somehow, e.g., 

```go
if btcDel.HasCovenantQuorums(params.CovenantQuorum) {
    if btcDel.HasInclusionProof() {

    } else {
        
    }
}
```

Reply:
hmm this is probably matter of preference but I try do avoid deeply nested `if/else` blocks.

Though I will experiment with this a bit to check the readability.

Reply:
hmm it looks good. I can leave with one level of nesting in this case 😅